solarwinds_apm/tracer_provider.py

```diff
@@ -11,6 +11,12 @@
 
 
 class SolarwindsTracerProvider(TracerProvider):
+    def get_sampler(self):
+        """
+        Returns the current sampler.
+        """
+        return self.sampler
```

Comment:
Since it's not `self._sampler` in parent class, then it would be fine to call `sampler = tracer_provider.sampler` instead of `sampler = tracer_provider.get_sampler()`. But an explicit getter is nice so this is fine.

Reply:
Sure. Done